Misc/NEWS.d/next/Documentation/2020-02-18-18-37-07.bpo-39572.CCtzy1.rst

```diff
@@ -0,0 +1 @@
+Updated _total_ attribute of TypeDict.
```

Comment:
There's an attribute `__total__`, which your docs do not mention (nor does the PEP, so it needn't be mentioned), and a class "flag" (not "attribute") named `total`. We don't use the convention `_word_` much for emphasis, so I recommend using either single quotes (`'total'`) or backticks (\`total`\) here.